wiki/installing-and-updating-drivers.md

```diff
@@ -85,7 +85,17 @@ See [Reinstalling GPU drivers from the OEM's website](reinstalling-gpu-drivers#m
 
 ## Troubleshooting 🔧
 ### If you can access Windows:
-1. Open the Device Manager by pressing <kbd>Win</kbd> + <kbd>X</kbd> and then clicking the "Device Manager" option. You can also search for it in the Start Menu.
+#### Verifying Driver Signatures
+::: Why you should do that?
```

Comment:
I don't think that's how you're supposed to use tip notes

Reply:
Personally, I think this should be in a paragraph of its own, not a tip note.

Reply:
OK, let me fix it now.